source/core/src/main/com/csse3200/game/entities/factories/EnemyFactory.java

```diff
@@ -95,7 +95,7 @@ public static Entity createFrog(Entity target) {
     animator.addAnimation("float", 0.1f, Animation.PlayMode.LOOP);
 
     frog
-            .addComponent(new CombatStatsComponent(config.health, 0, config.baseAttack, 0, 0, 0))
+            .addComponent(new CombatStatsComponent(config.health, 0, config.baseAttack, 0, 0, 0, false))
```

Comment:
Most of these files have been changed to have config files - you should include the rest of these magic numbers in those config files.